site/kubernetes/kube-addon.md

```diff
@@ -303,6 +303,10 @@ UDP connection from 10.32.0.7:56648 to 10.32.0.11:80 blocked by Weave NPC.
   pods can talk.
 - If you do set the `--cluster-cidr` option on kube-proxy, make sure
   it matches the `IPALLOC_RANGE` given to Weave Net (see below)
+- IP forwarding must be enabled on each node, in order for pods to
+  access Kubernetes services or other IP addresses on another
+  network. Check this with `sysctl net.ipv4.ip_forward`; the result
+  should be `1`.
```

Comment:
Maybe add a sentence "However, please be aware of security implications if enabling it".

Reply:
Added a simple note.